tst/ProducerApiTest.cpp

```diff
@@ -298,15 +279,10 @@ TEST_F(ProducerApiTest, create_produce_start_stop_stream)
 
             std::stringstream strstrm;
             strstrm << " TID: 0x" << std::hex << GETTID();
-            LOG_DEBUG("Putting frame for stream: " << kinesis_video_stream->getStreamName()
-                                                   << strstrm.str()
-                                                   << " Id: " << frame.index
+            LOG_DEBUG("Putting frame for stream: " << kinesis_video_stream->getStreamName() << strstrm.str() << " Id: " << frame.index
                                                    << ", Key Frame: "
-                                                   << (((frame.flags & FRAME_FLAG_KEY_FRAME) == FRAME_FLAG_KEY_FRAME)
-                                                       ? "true" : "false")
-                                                   << ", Size: " << frame.size
-                                                   << ", Dts: " << frame.decodingTs
-                                                   << ", Pts: " << frame.presentationTs);
+                                                   << (((frame.flags & FRAME_FLAG_KEY_FRAME) == FRAME_FLAG_KEY_FRAME) ? "true" : "false")
+                                                   << ", Size: " << frame.size << ", Dts: " << frame.decodingTs << ", Pts: " << frame.presentationTs);
```

Comment:
I think we should exclude the tests from the clang format check, the version before here looks better than how it formatted it.